### DISCUSSION TITLE
Questions about Poem project

### DISCUSSION THREAD

**Discussion by yardevx** [poem-web/poem#390] [General]
Hey guys, 
have a couple of question about Poem
How to annotate fields of structure with examples ? I'm mean like this
![openapi](https://user-images.githubusercontent.com/103759817/190230491-e2a7db95-5127-4fdb-ae68-10faf9ec863c.png)
and another question.
Can I set my own error message for validator? Like in example below
```
Create user schema
[derive(Debug, Object, Clone, Eq, PartialEq)]
struct User {
    /// Name
    [oai(validator(max_length = 64, message = "Sorry your name is too long. Pls try again"))]
    name: String
}
```

**Comment by sunli829** [comment]
You can use the `example` attribute to indicate that this object has an example, and then implement the `Example trait` for it.

https://github.com/poem-web/poem/blob/302f8a48f678432ac236048bfbd769ce00d2c3e3/poem-openapi/tests/object.rs#L762

**Reply by yardevx** [reply]
@sunli829 thank you for help ! It works well
